### DISCUSSION TITLE
[General] Wrong version released for test-node?

### DISCUSSION THREAD

**Discussion by enochSatis** [zkSync-Community-Hub/zksync-developers#248] [General] [answered]
### Environment

Localnet

### Provide a brief description of the functionality you're trying to implement.

Saw this Tweet and would like to try out the new zkSync Era test node (for the `--show-storage-logs=paid` flag to show `pubdata` used):
https://twitter.com/zksyncdevs/status/1737544399595196858?s=67&t=2ZdlQJIegio3Ld6LV5e5jA

### What is the specific issue or error you're encountering?

The version should be `Release: v0.1.0-alpha.14` (as stated in the gitHub link): https://github.com/matter-labs/era-test-node/releases/tag/v0.1.0-alpha.14

When I downloaded the Mac aarch64 version and ran:
```
./era_test_node --version
```
I get older version instead:
```
era_test_node 0.1.0-alpha.12
```

### Can you share the error messages or logs you're receiving, if any?

1. Download by `wget "https://github.com/matter-labs/era-test-node/releases/download/v0.1.0-alpha.14/era_test_node-v0.1.0-alpha.14-aarch64-apple-darwin.tar.gz"`
2. Run `./era_test_node --version`
3. Expected: `era_test_node 0.1.0-alpha.14`
4. What I get: `era_test_node 0.1.0-alpha.12`

### Have you made any recent changes to the contract before encountering this issue?

NA

### Are there any external libraries or contracts that your contract interacts with?

NA

### Can you provide the relevant portions of your contract code where the issue is occurring?

NA

### Have you tried to isolate the problem, and if so, what were the results?

NA

### What steps have you already taken to try to resolve the issue?

NA

**Comment by MexicanAce** [comment, marked as answer]
Hi @enochSatis !

Indeed, it looks like the version in the `Cargo.toml` is out-of-date and being updated now : https://github.com/matter-labs/era-test-node/pull/250

**Comment by dougtende** [comment]
It seems like you've encountered a discrepancy between the expected and actual version of the zkSync Era test node. This could be due to a variety of factors, including issues with the download process, caching problems, or mismatches in the release naming conventions.

To resolve this issue, you can try the following steps:
1. Ensure that you've downloaded the correct version of the zkSync Era test node for your platform (Mac aarch64).
2. Double-check the download process to ensure that the correct file was retrieved and extracted.
3. If the problem persists, you might consider reaching out to the zkSync development team for assistance or checking for any updates or announcements related to the test node version.

**Comment by go4comdathern** [comment]
Issue encountered with zkSync Era test node version. Downloaded v0.1.0-alpha.14, but getting v0.1.0-alpha.12 instead. Investigating discrepancy.

**Comment by lynnrosado** [comment]
Please ensure that you have downloaded the correct version of the zkSync Era test node (v0.1.0-alpha.14) for your system architecture and operating system. If the issue persists, you may want to reach out to the zkSync team for further assistance.

**Comment by tcaniao** [comment]
It seems like there might be an issue with the downloaded version of the zkSync Era test node. Please try downloading the correct version again or reach out to the zkSync team for assistance.

**Comment by crissshawn** [comment]
The user is attempting to try out the new zkSync Era test node with the --show-storage-logs=paid flag. They encountered an issue where the version of the downloaded Mac aarch64 version is not the expected version. They expected to see era_test_node 0.1.0-alpha.14 when running ./era_test_node --version, but instead got era_test_node 0.1.0-alpha.12. No error messages or logs were provided, and there are no recent changes to the contract or interactions with external libraries or contracts. The user has not attempted to isolate the problem or taken any steps to resolve the issue. No repository link was provided.
